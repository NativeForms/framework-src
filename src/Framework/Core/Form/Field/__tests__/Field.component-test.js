import React from 'react';
import { shallow } from 'enzyme';
import FieldComponent from '../Field.component';

describe('components <FieldComponent />', () => {
  it('renders text input component correctly', () => {
    const props = {
      type: 'text',
      attributes: {
        label: 'text input test'
      }
    };
    const wrapper = shallow(
      <FieldComponent {...props} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
