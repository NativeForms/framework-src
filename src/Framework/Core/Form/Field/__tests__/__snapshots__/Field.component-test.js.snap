// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <FieldComponent /> renders text input component correctly 1`] = `
<View>
  <View>
    <InjectIntl(TextInputComponent)
      label="text input test"
    />
  </View>
</View>
`;
